workspaces/zephyr/zephyr-master/zephyr-master.pts

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <project>
-  <qdid>165041</qdid>
-  <name>zephyr-master</name>
+  <qdid>175486</qdid>
+  <name>Zephyr-1</name>

```

Comment:
Not a big deal but let's keep it as `zephyr-master`.

Reply:
fixed